init.lua

```diff
@@ -135,8 +136,31 @@ function lib_mount.attach(entity, player, is_passenger)
 		if not entity.passenger_eye_offset then
 			entity.passenger_eye_offset = {x=0, y=0, z=0}
 		end
+
+		-- Support for more passengers
+		if not entity.passenger2_attach_at then
```

Comment:
Should be a dynamic array, not static members?

Reply:
Can you please be more precise?

Reply:
You shouldn't make members with numbers in it. Instead, you should make an array.

`entity.passenger[2].attach_at`

Reply:
Oh, thanks for explanation.
Will update and test it. 🙂

Reply:
Yes, please use arrays here - it would also reduce the code by a lot.

Reply:
> Yes, please use arrays here - it would also reduce the code by a lot.

I don't know exactly how to do that.
I have tried several ways, none of them work.